src/htr2hpc/train/slurm.py

```diff
@@ -51,6 +51,41 @@ def segtrain(
 
     logger.debug(f"segtrain command: {segtrain_cmd}")
     return segtrain_slurm.sbatch(segtrain_cmd)
+
+
+def recognition_train(
+    input_data_dir: pathlib.Path,
+    input_model: pathlib.Path,
+    output_model: pathlib.Path,
+    num_workers: int = 8,
+    # optional param to specify name based on document? include date?
+) -> int:
+    """Run ketos recognition training as a slurm job.
+    Returns the slurm job id for the queued job."""
+    recogtrain_slurm = Slurm(
+        nodes=1,
+        ntasks=1,
+        cpus_per_task=num_workers,
+        mem_per_cpu="2G",
+        gres=["gpu:1"],
+        job_name="train",
+        output=f"train_{Slurm.JOB_ARRAY_MASTER_ID}.out",
+        time=datetime.timedelta(minutes=20),
+        # time=datetime.timedelta(hours=2),
+    )
+    recogtrain_slurm.add_cmd("module purge")
+    recogtrain_slurm.add_cmd("module load anaconda3/2024.6")
+    recogtrain_slurm.add_cmd("conda activate htr2hpc")
+    logger.debug(f"sbatch file\n: {recogtrain_slurm}")
+    # sbatch returns the job id for the created job
+    recogtrain_cmd = (
+        f"ketos train -u NFD --augment --lag 50 --resize new -i {input_model}"
+        + f" -o {output_model} --workers {num_workers} -d cuda:0 "
+        + f"-f binary {input_data_dir}/train.arrow "
```

Comment:
```suggestion
        f"ketos train --resize union -i {input_model}"
        + f" -o {output_model} --workers {num_workers} -d cuda:0 "
        + f"-f binary {input_data_dir}/train.arrow "
```

I'm reviewing the [settings for transcription training in eScriptorium](https://gitlab.com/scripta/escriptorium/-/blob/develop/app/apps/core/tasks.py?ref_type=heads#L500), most of which are coming from [kraken's default specs](https://github.com/mittagessen/kraken/blob/main/kraken/lib/default_specs.py#L80). 

I think we can remove the `--augment` flag, since eScriptorium leaves that as kraken's default `False`. I would also remove the `-u NFD` flag, since the eScriptorium settings also leave normalization as the default `None`. It also has `--lag` kept as the default `10` instead of `50`. eScriptorium explicitly sets `--resize` as `union`.

Removing the `--augment` flag also means we do not *need* to add the `albumentations` package to our conda environment, though we will want it if we ever turn (an option for) kraken augmentation back on.

Reply:
Thanks for suggesting and explaining.